source/MaterialXContrib/MaterialXNode/MaterialXShadingNodeImpl.cpp

```diff
@@ -225,8 +225,10 @@ void MaterialXShadingNodeImpl<BASE>::updateShader(MHWRender::MShaderInstance& sh
 
     // Bind environment lighting
     // TODO: These should be options
-    std::string envRadiancePath = "san_giuseppe_bridge.hdr";
-    std::string envIrradiancePath = "san_giuseppe_bridge_diffuse.hdr";
+    static const std::string
```

Comment:
This shouldn't be static as it can change based on a user option but we have no mechanism to set this yet.
The option is per a scene global.

Reply:
Well, "static" reflects the current usage. When we have the mechanism to make this configurable, we won't forget to remove "static" :-)